modules/cco-ccoctl-configuring.adoc

```diff
@@ -115,7 +119,37 @@ $ chmod 775 ccoctl
 ----
 
 .Verification
+ifndef::ibm-power-vs[]
```

Comment:
As far as I know, this command and output should not have any conditions set. The same output applies to all clouds, the user will just select the one they are using. I think you need to remove this addition and add `powervs` to the list with the others.